src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const styleStringToArr = sourceString.trim().split(';');
+
+  for (const key of styleStringToArr) {
+    const styleToArr = key.trim().split(':');
+
+    if (styleToArr[1] && styleToArr[0] !== undefined) {
```

Comment:
The condition `styleToArr[1] && styleToArr[0] !== undefined` can be simplified. Since you're checking for the existence of both elements, you can use `styleToArr.length === 2` to ensure that both the property and value are present.